e3db/client.py

```diff
@@ -1409,3 +1409,55 @@ def read_file(self, record_id, destination_filename):
             version=response_json['meta']['version'],
             plain=response_json['meta']['plain']
         )
+
+    def write_note(self, data: dict, recipient_encryption_key: str, recipient_signing_key: str, options: NoteOptions):
+        """
+        Public Method to make a note, encrypt it locally, and send it
+        to the server.
+
+        Parameters
+        ----------
+        data : dict 
+
+
+        recipient_encryption_key : str
+
+
+        recipient_signing_key : str
+
+
+        options : types.NoteOptions
+
+
+        Returns
+        -------
+        e3db.Note
+            Decrypted E3DB note
+        """
+        # Do we need a KeyPair object? 
```

Comment:
I'd say either way is fine, but to go all the way whichever way you choose. E.g. if you are typing this to make it safer and clearer for the reader and next developer (including yourself), then make separate types for SigningKeyPair and EncryptionKeyPair, consider having those init methods validate the length / format of the public and private keys, and consider having the init method of the sdk client itself create these key pairs object (one time vs every time this method is called)

Reply:
I second have SigningKeypair and EncryptionKeyPair types, not having it in other places is an annoyance